app/src/main/java/com/islaharper/dawnofandroid/presentation/screens/login/LoginContent.kt

```diff
@@ -0,0 +1,105 @@
+package com.islaharper.dawnofandroid.presentation.screens.login
+
+import android.content.res.Configuration
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.LinearProgressIndicator
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.islaharper.dawnofandroid.R
+import com.islaharper.dawnofandroid.domain.model.MessageBarState
+import com.islaharper.dawnofandroid.presentation.common.GoogleButton
+import com.islaharper.dawnofandroid.presentation.components.MessageBar
+
+@Composable
+fun LoginContent(
```

Comment:
Minor:
Can these two composables be `private` inside `LoginScreen.kt`? It helps the reader understand the context and it doesn't make `LoginScreen` excessively long.

Reply:
Refactored, in next commit